src/iomei.cpp

```diff
@@ -8264,6 +8317,13 @@ bool MEIInput::ReadXMLComment(Object *object, pugi::xml_node element)
     return true;
 }
 
+bool MEIInput::IsAnnotScore(pugi::xml_node annot)
+{
+    // If the annotation is a score annotation, it'll have @type="score" (we can also guess)
+    std::string value = annot.attribute("type").value();
+    return value == "score";
```

Comment:
Stylistic preference
```suggestion
    return (value == "score");
```